libraries/standard/http/test/Makefile

```diff
@@ -6,7 +6,7 @@ SOURCE = ../src
 #CFLAGS = std=c99 -Wall -fprofile-arcs -ftest-coverage -O0 -ggdb
 
 # include directories given to the compiler (default: none)
-INCLUDE = ../include
+INCLUDE = . ../include ../../../.. ../../utilities/include
```

Comment:
Storing the root directory path in a variable can make these more readable
```
ROOT_PATH=../../
INCLUDE=$ROOT_PATH/libraries/standard/utilities/include <remaining-path>
```

Reply:
sure